content/Secure_Domain/Introduction.adoc

```diff
@@ -0,0 +1,8 @@
+== Introduction
+
+YubiKeys with firmware version 5.3.0 and later support Secure Channel Protocol 03 (SCP03) for establishing secure communication channels. Starting with firmware version 5.7.2, they also support Secure Channel Protocol 11 (SCP11). For both protocols, secure communication with the YubiKey requires the secure channel to be authenticated using the appropriate certificate chain. Detailed information about the implementation specifics of these protocols can be found in Yubico's technical documentation here.
+
+In SCP11 mode, the certificate stored on the YubiKey should be validated against the full certificate chain. All genuine YubiKeys with firmware version 5.7.4 or later are issued with Secure Domain certificates. These certificates are linked to the Root Certificate Authority (Root CA) through intermediate certificates, ensuring the integrity and trustworthiness of the YubiKey.
```

Comment:
This page doesn't have a link to `yubico-ca-certs.txt`, it probably should so people can find the more in-depth usage instructions.

```suggestion
In SCP11 mode, the certificate stored on the YubiKey should be validated against the full certificate chain. All genuine YubiKeys with firmware version 5.7.4 or later are issued with Secure Domain certificates. These certificates are linked to the link:/PKI/yubico-ca-certs.txt[Root Certificate Authority (Root CA)] through intermediate certificates, ensuring the integrity and trustworthiness of the YubiKey.
```